DOCUMENTATION.md

```diff
@@ -205,7 +209,9 @@ with open('file.pdf', 'rb') as f:
     }
 ```
 
-Contact the GOV.UK Notify team on the support page or through the Slack channel to enable this function for your service.
+####Add a link to the email template
+
+To include a link to the document in your email notification, add a placeholder to the template in Notify.
```

Comment:
What sort of placeholder? Is there any particular syntax that must be used?

Reply:
It's the ((double brackets)) placeholder used for personalisation in the template.
There will be formatting guidance on the Edit template screen, but I'd like to include an example here too. Need to confirm exactly what the placeholder should be first.